scaleway/resource_k8s_pool_beta.go

```diff
@@ -228,11 +286,18 @@ func resourceScalewayK8SPoolBetaUpdate(d *schema.ResourceData, m interface{}) er
 		updateRequest.Size = scw.Uint32Ptr(uint32(d.Get("size").(int)))
 	}
 
-	_, err = k8sAPI.UpdatePool(updateRequest)
+	res, err := k8sAPI.UpdatePool(updateRequest)
 	if err != nil {
 		return err
 	}
 
+	if d.Get("wait_for_pool_ready").(bool) { // wait for nodes to be ready if specified
```

Comment:
`// wait for the pool to be ready if specified` ? 

Either change comment or change name of the field.

Reply:
And same as above :)

Reply:
modified